11/debian/bullseye-slim/hotspot/Dockerfile

```
@@ -11,7 +11,7 @@ RUN jlink \
 FROM debian:bullseye-20220509-slim
 
 RUN apt-get update \
-  && apt-get install -y \
+  && apt-get install -y --no-install-recommends \
```

```suggestion
  && apt-get install -y --no-install-recommends \
      ca-certificates \
```

Ca Cert is suggested by `curl` but not installed. It is needed for the entrypoint shell scripts.

---

You mean by the `curl(1)` invocation `install-plugin.sh`? I suppose so.

---

Pardon my non-culture but, what is hte meaning of the `1` beetween parenthesis in the `curl(1)`?

---

many thanks for the pointer and explanation.